classes/book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+  attr_accessor :title, :author, :rentals
+
+  def add_rental(rental)
+    @rentals << rental unless @rentals.include?(rental)
+  end
```

Comment:
- [ ] I think the parameters to pass to the `add_rental` method should be `person` and `date`. Follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) in the previous lesson to get and understand more.
`def add_rental(person, date)`

- [ ] The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
`Rental.new(date, self, person)`